Cargo.lock

```diff
@@ -25,9 +19,9 @@ checksum = "f26201604c87b1e01bd3d98f8d5d9a8fcbb815e8cedb41ffccbeb4bf593a35fe"
 
 [[package]]
 name = "aead"
-version = "0.5.1"
+version = "0.5.2"
```

Comment:
:wrench: I was hoping most of the cargo lock would get fixed by the `tempdir` removal but there are still 1600+ changes.
Should move this closer to the mainline, something like the following could be done with this branch checked out.
```shell
git checkout origin/master Cargo.lock
cargo check
git add Cargo.lock
git commit -m "Revert unnecessary Cargo.lock changes"
```